src/libraries/Microsoft.Extensions.DependencyInjection.Specification.Tests/src/Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj

```diff
@@ -15,6 +15,11 @@
     <PackageDescription>Suite of xUnit.net tests to check for container compatibility with Microsoft.Extensions.DependencyInjection.</PackageDescription>
     <!-- this assembly doesn't need to be binplaced -->
     <EnableBinPlacing>false</EnableBinPlacing>
+    <!-- This is warning in net8.0 because xunit.assert pulls in netstandard packages that have non-portable RIDs (OpenSsl).
+         They won't actually get used at run time - package conflict resolution would normally handle removing them. However,
+         due to the way the repo builds, the platform manifest for the ref package has not yet been generated when this test
+         is being built, so they are not removed (and the SDK warns on it). -->
+    <NoWarn>$(NoWarn);NETSDK1206</NoWarn>
```

Comment:
Can we just reference a later NETStandard.Library to avoid those dependencies?  The latest NETStandard.Library package won't bring in the old PackageReferences.

Reply:
Yes, please add a PackageRefernce to NetStandard.Library with PrivateAssets="all"

Reply:
Can you check https://github.com/dotnet/runtime/pull/88827/commits/2adf3d901d4691c35f8966d6ecae1160cc1609c1 is what you mean?